Projects/l0/Controller_motor/lib/Controller_motor/controller_motor.c

```diff
@@ -395,14 +394,12 @@ static void ControllerMotor_MsgHandler(container_t *container, msg_t *msg)
         if (msg->header.size == sizeof(asserv_pid_t))
         {
             // fill the message infos
-            if ((motor.mode.mode_angular_position || motor.mode.mode_linear_position) &&
-                !(motor.mode.mode_angular_speed || motor.mode.mode_linear_speed))
+            if ((motor.mode.mode_angular_position || motor.mode.mode_linear_position) && !(motor.mode.mode_angular_speed || motor.mode.mode_linear_speed))
             {
                 // only position control is enable, we can save PID for positioning
                 memcpy(&position, msg->data, msg->header.size);
             }
-            if ((motor.mode.mode_angular_speed || motor.mode.mode_linear_speed) &&
-                !(motor.mode.mode_angular_position || motor.mode.mode_linear_position))
+            if ((motor.mode.mode_angular_speed || motor.mode.mode_linear_speed) && !(motor.mode.mode_angular_position || motor.mode.mode_linear_position))
```

Comment:
Perhaps we should put OR and AND operators at line beginning.  So, after clang format processing, the conditions should be on different lines.